docs/design/v2/operator-filtering-study.md

```diff
@@ -0,0 +1,139 @@
+* constructs a filtered FBC based on the filtered content
+* compares that filtered FBC to the filtered FBC from the previous mirroring: in order to take into account only the delta images.
+
+This implementation is not ideal for users, because :
+* the mirrored bundles don't always allow a smooth upgrade of operators
+* some mirrored bundles are sometimes missing for certain imageSetConfig use-cases
+* some mirrored bundles are mirrored although not part of the upgrade graph expected by OLM : Volume of mirrored content unnecessarily high
+
+## Solutions studied for filtering in V2
+
+This lead the oc-mirror team to implement a different approach to filtering operator catalogs for the V2.
+
+When selecting bundles for mirroring, oc-mirorr V2 will not verify if the bundles were previously mirrored. The list of bundles selected will be solely based on processing the FBC declarative config of the catalog with the filtering from the imageSetConfig. The exclusion of some bundles from the final mirroring will be differed to another component of oc-mirror, which will handle incremental mirroring (archive diff). 
+
+When selecting bundles for mirroring, oc-mirror V2 will not attempt to bridge the graph between 2 channels, or between one catalog and the other. It will strictly respect the client's instructions given by the imageSetConfig.
+
+Regarding the way to determine the list of bundles included in the filtering. several solutions were studied, described below.
+
+For a complete list of use cases that are accounted for, please see [below](#acceptance-criteria-set-for-v2).
+
+:warning: In this document, we do not discuss if/how the catalog would be rebuilt with the filtered content. We are separating the two discussions, even though we are aware that according to the solution chosen here, the filtered FBC catalog might not be considered as valid from the standpoint of OLM, or might not allow an operator to be upgraded correctly with such a filtered FBC catalog.
+
+### S1 - SemVer
+
+Most bundle versions are semver compatible. The bundles included in a channel can simply be sorted by version and any bundles that fall in the range selected in the ImageSetConfig are taken. 
+
+Such an algorithm will result in idempotent results compared to V1 results. Nevertheless, the upgrade graph information conveyed by the `replaces`, `skip` and `skipRange` is not taken into account. 
+This algorithm is totally different from the one used by OLM. There is no guarantee here that the selected list of bundles would allow OLM to perform an upgrade for the selected operator. There may also be a much shorter path between the minVersion and the maxVersion requested. oc-mirror will, in this case, mirror more bundles than what a cluster would need to perform the upgrade. 
+
+### S2 - Call `render-graph`
```

Comment:
I honestly thought that we were not going to have a dependency on opm render-graph - as per Jordan Keister's suggestion in our discussion with  the OLM team

Reply:
I'm just saying this as it adds unnecessary complexity

Reply:
Agreed. In S2, we explain what the consequences would be if we were to call render-graph from oc-mirror, and explain why this is not a viable solution. 

Are you saying that S2 should not be presented and/or be presented same as S4?